ctapipe/image/muon/features.py

```diff
@@ -55,15 +52,14 @@ def photon_ratio_inside_ring(
         width of the ring
     """
 
-    total = np.sum(weights)
-
     pixel_r = np.sqrt((center_x - pixel_x)**2 + (center_y - pixel_y)**2)
     mask = np.logical_and(
         pixel_r >= radius - 0.5 * width,
         pixel_r <= radius + 0.5 * width
     )
```

Comment:
Why such a narrow range?

Reply:
I think you are confusing something here.

going from -0.5 width to 0.5 width is the very definition of a ring centered at radius `r` with the corresponding width.

Reply:
The width is "half-width", I think. So it should be (-width, width). But there will be some light (~32% assuming gaussian) beyond that distance, so if the goal is to compare "ring light" with (non-ring) shower light perhaps a larger range would be better.

Reply:
In the tool, the width to be taken into account here is currently customizable in terms of the pixel diameter. The default is `width = 1.5 * pixel diameter`

Reply:
ok, I was assuming it is the fitted ring width. It is confusing that that the fitted one is 'half-width',  while this one is full width. But I agree it is not a problem of this function. One should just choose a 'width' value to call it depending on the goal, e.g. catching all of the muon light or not.

Reply:
I think we should probably not call the standard deviation of the gaussian profile `width`. It is not a width in any usual sense.